src/openrct2/network/Network.cpp

```diff
@@ -3234,11 +3402,16 @@ bool network_is_desynchronised()
     return gNetwork.IsDesynchronised();
 }
 
-void network_check_desynchronization()
+bool network_check_desynchronization()
```

Comment:
While we're at it, we can change this to `network_check_desynchronisation()` for consistency with everything else.